crates/processing_render/src/render/mod.rs

```diff
@@ -8,14 +8,14 @@ use command::{CommandBuffer, DrawCommand};
 use material::MaterialKey;
 use primitive::{TessellationMode, empty_mesh};
 
-use crate::{Flush, SurfaceSize, image::PImage, render::primitive::rect};
+use crate::{Flush, graphics::SurfaceSize, image::Image, render::primitive::rect};
 
 #[derive(Component)]
 #[relationship(relationship_target = TransientMeshes)]
-pub struct BelongsToSurface(pub Entity);
+pub struct BelongsToGraphics(pub Entity);
```

Comment:
as we start to solidify all of this, it could be nice to have a diagram outlining the relationships of our bevy entities